helm/observability-operator/files/crds/organization.giantswarm.io_org.yaml

```diff
@@ -0,0 +1,96 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
```

Comment:
Did you create the yaml yourself ? Because you should create CRDS from go code and générate the yaml 😅

Reply:
<img src="https://media2.giphy.com/media/WrNfErHio7ZAc/giphy.gif"/>

Reply:
Putting this PR in draft then :sweat_smile: